VMWare/vmware-vcenter/ingest/parser.yml

```
@@ -0,0 +1,94 @@
+          source.user.name: "{{parsed_event.message.source_user_name}}"
+          user_agent.original: "{{parsed_event.message.user_agent}}"
+          host.name: "{{parsed_event.message.hostname}}"
+          event.reason: "{{parsed_event.message.reason}}"
+          log.level: "{{parsed_event.message.log_level}}"
+          http.request.method: "{{parsed_event.message.http_method}}"
+          url.path: "{{parsed_event.message.url_path}}"
+          http.response.status_code: "{{parsed_event.message.response_status}}"
+          http.response.bytes: "{{parsed_event.message.response_bytes}}"
+          vmware_vcenter.event_id: "{{parsed_event.message.id}}"
+          event.code: "{{parsed_event.message.event_code}}"
+          process.pid: "{{parsed_event.message.pid}}"
+          process.end: "{{parsed_event.message.process_time}}"
+          process.name: "{{parsed_event.message.process}}"
+          source.port: "{{parsed_event.message.source_port}}"
+          http.version: "{{parsed_event.message.http_version}}"
+          destination.port: "{{parsed_event.message.destination_port}}"
+          source.address: "{{parsed_event.message.source_address}}"
+          client.geo.country_iso_code: "{{parsed_event.message.locale}}"
+          host.ip: "{{parsed_event.message.ip_address}}"
+          vmware_vcenter.network.port: "{{parsed_event.message.local_port1}}"
+          vmware_vcenter.login_time: "{{parsed_event.message.login_time}}"
+          vmware_vcenter.api_invocations: "{{parsed_event.message.api_invocations}}"
+          vmware_vcenter.datetime: "{{parsed_event.message.datetime}}"
+          vmware_vcenter.upstream_status: "{{parsed_event.message.upstream_status}}"
+          vmware_vcenter.host.ip2: "{{parsed_event.message.local_ip2}}"
+          vmware_vcenter.network.port2: "{{parsed_event.message.local_port2}}"
+          vmware_vcenter.conn_status: "{{parsed_event.message.conn_status}}"
+          vmware_vcenter.commit_time: "{{parsed_event.message.commit_time}}"
+          vmware_vcenter.thread: "{{parsed_event.message.thread}}"
+          vmware_vcenter.operationID: "{{parsed_event.message.operationID}}"
+
+      - set:
+          observer.vendor: "VMWare"
+          observer.product: "VCenter"
+          event.kind: "event"
+
+      - translate:
+        dictionary:
+          vim.event.UserLoginSessionEvent: [ "start" ]
+          vim.event.BadUsernameSessionEvent: [ "end" ]
+          vim.event.UserLogoutSessionEvent: [ "end" ]
+          vim.event.AlreadyAuthenticatedSessionEvent: [ "end" ]
+        mapping:
+          parsed_event.message.event_code: event.type
+        fallback: [ "connection" ]
+
+      - translate:
+        dictionary:
+          vim.event.UserLoginSessionEvent: [ "start" ]
+          vim.event.BadUsernameSessionEvent: [ "end" ]
+          vim.event.UserLogoutSessionEvent: [ "end" ]
+          vim.event.AlreadyAuthenticatedSessionEvent: [ "end" ]
```

`start` and `end` are not [`event.type`](https://www.elastic.co/guide/en/ecs/8.6/ecs-allowed-values-event-category.html)
```suggestion
          vim.event.UserLoginSessionEvent: [ "authentication" ]
          vim.event.BadUsernameSessionEvent: [ "authentication" ]
          vim.event.UserLogoutSessionEvent: [ "authentication" ]
          vim.event.AlreadyAuthenticatedSessionEvent: [ "authentication" ]
```